packages/rendermime/src/renderers.ts

```diff
@@ -808,10 +821,16 @@ function renderTextual(
   // Unpack the options.
   const { host, sanitizer, source } = options;
 
-  // Create the HTML content.
-  const content = sanitizer.sanitize(Private.ansiSpan(source), {
-    allowedTags: ['span']
-  });
+  const ansiPrefixRe = /\x1b/; // eslint-disable-line no-control-regex
```

Comment:
We need to disable [`no-control-regex`](https://eslint.org/docs/latest/rules/no-control-regex) here, the same way we already disable it in `ansiSpan()`

https://github.com/jupyterlab/jupyterlab/blob/c9700a2802dabfb44109b4327473b6a593e7cc97/packages/rendermime/src/renderers.ts#L1560-L1561

because this invisible control character is exactly what we want to detect.